airflow/bin/cli.py

```diff
@@ -365,24 +364,22 @@ def run(args, dag=None):
         # tries to write to a log file created by the other user.
         log_base = os.path.expanduser(conf.get('core', 'BASE_LOG_FOLDER'))
         directory = log_base + "/{args.dag_id}/{args.task_id}".format(args=args)
+        # Create the log file and give it group writable permissions
+        # TODO(aoen): Make log dirs and logs globally readable for now since the SubDag
+        # operator is not compatible with impersonation (e.g. if a Celery executor is used
+        # for a SubDag operator and the SubDag operator has a different owner than the
+        # parent DAG)
         if not os.path.exists(directory):
             # Create the directory as group writable using custom mkdirs
             # as os.makedirs doesn't set mode properly.
             mode = 0x775
-            mkdirs(directory, 0o775)
+            mkdirs(directory, 0o777)
```

Comment:
Does it need to have these broad permissions. World readable is something else than world writable/executable. 

I won't be able to run the code as is in our environment (not allowed). It will be caught by security monitoring right away. 

Maybe we should make this change dependent on making backfills use the scheduler (ie that fixes the subdagoperator)?